arex-instrumentation-api/src/main/java/io/arex/inst/runtime/util/TypeUtil.java

```diff
@@ -386,7 +386,11 @@ private static String parameterizedTypeToString(ParameterizedType type) {
     public static String arrayObjectToString(Object[] objects) {
```

Comment:
Please remove this method and call the `ArrayUtils.toString` method directly, as follows

`ArrayUtils.toString((Object[]) object, o -> o.getClass().getTypeName())`